src/Workspaces/SharedUtilitiesAndExtensions/Workspace/VisualBasic/Extensions/ContextQuery/VisualBasicSyntaxContext.vb

```diff
@@ -54,6 +54,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions.ContextQuery
         Public ReadOnly IsPreprocessorEndDirectiveKeywordContext As Boolean
         Public ReadOnly IsWithinPreprocessorContext As Boolean
 
+        Public ReadOnly IsObjectCreationContext As Boolean
```

Comment:
Consider moving this up to `SyntaxContext`, since it now exists in both C# and VB